01-shapefile-attributes.Rmd

```diff
@@ -294,71 +284,78 @@ levels(lines_HARV$TYPE)
 summary(lines_HARV$TYPE)
 
 #plot the lines data, apply a diff color to each category
-plot(lines_HARV, col=lines_HARV$TYPE,
+plot(lines_HARV, 
+     col=lines_HARV$TYPE,
      lwd=3,
      main="Roads at the NEON Harvard Forest Field Site")
 ```
 
-
 ###Adjust Line Width
-We can adjust the width of our plot lines too using `lwd`. We can set all lines
+We can also adjust the width of our plot lines using `lwd`. We can set all lines
 to be thicker or thinner using `lwd=`. Or given we have a factor with 4 levels, 
 we can provide an vector of numbers, each of which represents the thickness of
 one of our four levels or categories as follows:
      
 ```{r adjust-line-width}
 #make all lines thicker
-plot(lines_HARV, col=lines_HARV$TYPE,
-     main="Roads at the NEON Harvard Forest Field Site",
+plot(lines_HARV, 
+     col=lines_HARV$TYPE,
+     main="Roads at the NEON Harvard Forest Field Site \n All Lines Thickness=6",
      lwd=6)
 
 levels(lines_HARV$TYPE)
 #adjust width of each level
 #in this case, boardwalk (the first level) is the widest.
-plot(lines_HARV, col=lines_HARV$TYPE,
-     main="Roads at the NEON Harvard Forest Field Site \n Boardwalk Line Width Wider",
-     lwd=c(10,2,3,4))
+plot(lines_HARV, 
+     col=lines_HARV$TYPE,
+     main="Roads at the NEON Harvard Forest Field Site \n Boardwalk Line Width Widest",
+     lwd=c(10,2,4,6))
 
 ```     
 
 ##Add Plot Legend
+We can add a legend to our plot too. When we add a legend, we use the following
+elements to specify labels and colors:
 
-We can add a legend to our plot too. When we add a legend, we can use
-the following elements to specify labels and colors:
-
-* `levels(lines_HARV$TYPE)` - Label the legend elements using the categories of 
-`levels` in our `TYPE` attribute (boardwalk, footpath, etc).
-* `fill=palette()` - apply unique colors to the boxes in our legend. 
-`palette()` is the default set of colors that `R` applies to all plots. 
-* We can specify the **location** of our legend too. Let's use `bottomright` for 
-this plot. We could also use `top`, `topright`, etc.
+* `bottomright`: We specify the **location** of our legend by using a default 
+keyword. We could also use `top`, `topright`, etc.
+* `levels(objectName$attributeName)`: Label the **legend elements** using the
+categories of `levels` in an attribute (e.g., levels(lines_HARV$TYPE) means use
+the levels boardwalk, footpath, etc).
+* `fill=`: apply unique **colors** to the boxes in our legend. `palette()` is 
+the default set of colors that `R` applies to all plots. 
 
 Let's add a legend to our plot.
 
 ```{r add-legend-to-plot }
-plot(lines_HARV, col=lines_HARV$TYPE,
+plot(lines_HARV, 
+     col=lines_HARV$TYPE,
      main="Roads at the NEON Harvard Forest Field Site\n Default Legend")
 #add a legend to our map
-legend("bottomright", NULL, levels(lines_HARV$TYPE), fill=palette())
+legend("bottomright",   #location of legend     
+       NULL,            # 
```

Comment:
We don't specify why the NULL is here.  It plots without the NULL as well.  Add comment?